src/components/footer.js

```diff
@@ -13,25 +14,18 @@ import logoImg from "../images/logos/cto_logo_horizontal_white.png";
 function Footer() {
   const { t } = useTranslation();
   const primaryLinks = ["/link-one", "/link-two", "/link-three", "/link-four"];
-  const socialLinks = {
-    facebook: "www.facebook.com",
-    twitter: "www.twitter.com",
-    youtube: "www.youtube.com",
-    instagram: "www.instagram.com",
-    rss: "https://www.google.com/search?q=rss",
-  };
-  const socialLinkTypes = [
-    "facebook",
-    "twitter",
-    "youtube",
-    "instagram",
-    "rss",
-  ];
 
   return (
     <FooterUSWDS
       size="medium"
-      // returnToTop={returnToTop}
+      returnToTop={
+        <GridContainer>
+          <Link to="#top" className="usa-footer__return-to-top">
+            Return to top
+          </Link>
+          {/* TODO: for some reason class name has to be manually added*/}
```

Comment:
What do you mean by this? That you need to add `className="usa-footer__return-to-top"` to see the proper styling?

Reply:
Yes, it's weird because there's a `returnToTop` attribute that automatically includes the class, but it doesn't here. It works in LPaaS 1.0, but I cannot see how they value that attribute. I think it's its own component, but it still does not include the class when I have it as a separate component.

Reply:
Oh interesting!